cmake/multilib.yaml.in

```diff
@@ -135,6 +135,21 @@ Mappings:
   Flags:
   - --target=armv5e-unknown-none-eabihf
 
+# armv7ve is a GCC name for v7-A with the virtualisation extension, for library
```

Comment:
This will definitely work. Just wondering if there's a way to use a regular expression to match all 4 in one as they all map to `armv7-unknown-none-eabi`.

Possibly (untested)
```
- Match: --target=(arm|thumb)v7ve-unknown-none-(eabi|eabihf)
- Flags:
  - --target=armv7-unknown-none-eabihf
```

Reply:
Thanks, that does simplify things, but I think we still need two `Match` blocks because the ABI needs to match in the input and output.

Reply:
Yes, I forgot about that part. I can remember wishing that we could cite part of the regex match in the flags, something like --target=armv7-unknown-none-$2 to inappropriately use my memory of Perl for the second match in the RE.

Perhaps a future multilib improvement if we find it cropping up a lot.